Sources/YouboraConfig.swift

```diff
@@ -9,34 +9,59 @@ import YouboraLib
 
 struct YouboraConfig: Decodable {
     let accountCode: String
+    
     let username: String?
+    let userEmail: String?
+    let anonymousUser: String?
     let userType: String?
     let obfuscateIP: Bool?
+    let userObfuscateIp: Bool?
+    
     var httpSecure: Bool? = true
     let contentCDN: String?
+    let autoDetectBackground: Bool?
+    let isOffline: Bool?
     
     let media: Media?
+    let content: Content?
     let ads: Ads?
     let properties: Properties?
+    
+    let customDimensions: CustomDimensions?
     let extraParams: ExtraParams?
     let houseHoldId: String?
+    let isAutoStart: Bool?
+    let isEnabled: Bool?
+    let isForceInit: Bool?
     let appName: String?
-    
+    let app: App?
     let parse: Parse?
+    let device: Device?
+    let network: Network?
+    let errors: Errors?
     
     func options() -> YBOptions {
         let options = YBOptions()
         
         options.accountCode = accountCode
+        
         options.username = username
+        options.userEmail = userEmail
+        options.anonymousUser = anonymousUser
         options.userType = userType
         options.userObfuscateIp = obfuscateIP != nil ? NSNumber(booleanLiteral: obfuscateIP!) : nil
+        options.userObfuscateIp = userObfuscateIp != nil ? NSNumber(booleanLiteral: userObfuscateIp!) : nil
```

Comment:
line 52 same as  53  
is it a typo for obfuscateIP?

Reply:
Actually this is legacy settings support, it was previously "obfuscateIP" and if you are using new name "userObfuscateIp" the new one will be set to options.userObfuscateIp